ipn/local.go

```diff
@@ -745,6 +745,17 @@ func (b *LocalBackend) FakeExpireAfter(x time.Duration) {
 	b.send(Notify{NetMap: b.netMap})
 }
 
+func (b *LocalBackend) Ping(ipStr string) {
+	ip, err := netaddr.ParseIP(ipStr)
+	if err != nil {
+		b.logf("ignoring Ping request to invalid IP %q", ipStr)
```

Comment:
Send as a PingResult.Err to respect the (currently implicit) contract that a Ping always gets a PingResult.

Reply:
It's not true that a Ping always get a PingResult. And this is an internal error: the client should've never sent this message if it's invalid. So dropping it (with a log) seems fine.